_inc/lib/tracks/class.tracks-client.php

```diff
@@ -184,6 +184,7 @@ static function get_connected_user_tracks_identity() {
 		}
 
 		return array(
+			'blogid' => Jetpack_Options::get_option( 'id' ),
```

Comment:
Are you planning to introduce any checks for the blog ID here? Because if we don't have it yet, this will fall back to `false`  and not `0` - I'd expect we always want to keep it a numeric value.

Reply:
as mentioned yesterday, yes, I was meaning to add a condition.